rules/link-users-by-email-with-metadata.md

```diff
@@ -0,0 +1,75 @@
+---
+gallery: true
+short_description: Link any accounts that have the same email address while merging metadata.
+categories:
+  - access control
+---
+
+# Link Accounts with Same Email Address while Merging Metadata
+This rule will link any accounts that have the same email address while merging metadata.
+
+```js
+function (user, context, callback) {
+  var request = require('request@2.56.0');
+  // Check if email is verified, we shouldn't automatically
+  // merge accounts if this is not the case.
+  if (!user.email_verified) {
+    return callback(null, user, context);
+  }
+  var userApiUrl = auth0.baseUrl + '/users';
+  
+  request({
+   url: userApiUrl,
+   headers: {
+     Authorization: 'Bearer ' + auth0.accessToken
+   },
+   qs: {
+     search_engine: 'v2',
+     q: 'email:"' + user.email + '" -user_id:"' + user.user_id + '"',
```

Comment:
Should we not use `email.raw` instead? I remember some issues with tokenization of query values leading to unexpected results.